lib/index.js

```diff
@@ -21,7 +21,7 @@ $.get(weather_url, function(data,status){
   //glance right
   $("#cityState").text(`${data["location"]}`)
   $("#date").text(date)
-  // need time
+
```

Comment:
I know that there is a way to iterate over the json that we get back in the response. I have not yet implemented this either, it will be a refactor